packages/proposal-module-adapter/adapters/cw-proposal-single/components/ProposalLine/Desktop.tsx

```diff
@@ -52,5 +60,28 @@ export const ProposalLineDesktop = ({ className }: BaseProposalLineProps) => {
       <p className="col-span-3 truncate link-text">{proposal.title}</p>
       <p className="text-right truncate body-text">{expirationString}</p>
     </div>
-  )
+    </Tooltip>) : (<div
+    className={clsx(
+      'grid grid-cols-6 items-center p-4 text-sm rounded-lg transition bg-primary hover:bg-secondary',
+      {
+        'bg-gray-500/30': msSinceUpdated < 24 * 60 * 60 * 1000,
```

Comment:
hey there's code duplication here, why don't we extract this so in the future changes can be made in one place.

Reply:
Yeah, can we just set the contents to a variable and then either return them wrapped with the Tooltip or not? Like `return proposalModuleVersion === ContractVersion.V0_2_0 ? <Tooltip label="...">{contents}</Tooltip> : contents`

Reply:
ok!